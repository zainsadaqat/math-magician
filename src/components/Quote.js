import React from 'react';

const Quote = () => {
  return (
    <section>
      <h2 className="heading">Quote of the Day</h2>
      <p className="paragraph">
        Mathematics is not about numbers, equations, computations, or
        algorithms: It is about understanding. -- William Paul Thurston
      </p>
    </section>
  );
};

export default Quote;
