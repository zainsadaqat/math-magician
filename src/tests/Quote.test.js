import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Matches Snapshot', () => {
  const tree = renderer
    .create(
      <Quote />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
