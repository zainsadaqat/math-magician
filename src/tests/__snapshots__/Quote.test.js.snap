// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot 1`] = `
<section>
  <h2
    className="heading"
  >
    Quote of the Day
  </h2>
  <p
    className="paragraph"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding. -- William Paul Thurston
  </p>
</section>
`;
